operations/research-and-development/product/release-process/desktop-release.md

```diff
@@ -2,22 +2,24 @@
 
 This document outlines the release process for the Mattermost desktop app.
 
-Desktop App releases follow a fixed schedule of 4 releases per year. Releases ship quarterly by the 16th of March, June, September, and December of each year.
+Desktop app releases follow a fixed schedule of 4 releases per year. Releases ship quarterly by the 16th of March, June, September, and December of each year.
 
 A dot release will be prepared sooner if [Electron](https://github.com/electron/electron/releases) releases a security update, or if other urgent bugs are found.
 
-Please refer to [the desktop app release playbook](https://community.mattermost.com/playbooks/playbooks/h3a39biacpnuim7ufmwiuuoxfo/outline) for a most up-to-date checklist.
+Please refer to [the desktop app release playbook](https://community.mattermost.com/playbooks/playbooks/h3a39biacpnuim7ufmwiuuoxfo/outline) for the release checklist.
 
 For the desktop app release build process, please refer to [this document](https://developers.mattermost.com/internal/desktop-release-process/).
 
 ## Release timeline
 
 Note: T-minus counts are measured in "working days" (weekdays other than major holidays concurrent in US and Canada) prior to release day.
 
+The Desktop app release schedule follows the cloud/self-hosted release schedule to align all release testing cycles.
```

Comment:
@devinbinnie For your review on desktop release changes.

Reply:
Looks right, we still only release every 3 months though so is this just so the days coordinate?
Might be worth putting the every 3 months bit in here as well :)

Reply:
Yes, mainly for days coordination. Sure, I think Amy added the 3 months bit on recent commits.